cudax/include/cuda/experimental/__execution/sequence.cuh

```diff
@@ -149,24 +149,23 @@ struct _CCCL_TYPE_VISIBILITY_DEFAULT __seq_t::__sndr_t
     return execution::get_env(__sndr2_);
   }
 
-  _CCCL_NO_UNIQUE_ADDRESS __seq_t __tag_;
+  _CCCL_NO_UNIQUE_ADDRESS sequence_t __tag_;
   _CCCL_NO_UNIQUE_ADDRESS __ignore __ign_;
```

Comment:
Is that ever used?

Reply:
Yes, when using a sender like this to initialize a structured binding, the sender's tag is always the first element. That makes it possible to build visitors for sender trees.

Reply:
🙀